hbp_knowledge/pp2a/seshacharyulu2013.bel

```diff
@@ -203,80 +242,118 @@ a(CHEBI:cantharidin) -| act(p(FPLX:PPP2))
 negative breast cancer cells [50]."
+SET Confidence = "Medium"
+SET MeSHDisease = "Breast Neoplasms"
 a(CHEBI:bortezomib) -| p(HGNC:CIP2A)
+a(CHEBI:bortezomib) -| r(HGNC:CIP2A)
 a(CHEBI:bortezomib) -> act(p(FPLX:PPP2C))
+UNSET MeSHDisease
+UNSET Confidence
 
 SET Evidence = "Methylation [52] and phosphorylation [53] are two major
 modifications that have been shown to modulate PP2A catalytic efficiency."
+SET Confidence = "High"
 p(FPLX:PPP2, pmod(M)) -- act(p(FPLX:PPP2))
 p(FPLX:PPP2, pmod(Ph)) -- act(p(FPLX:PPP2))
+UNSET Confidence
 
+#can be annotated with respect to shi2009 subunit classification
 SET Evidence = "Phosphorylation of Y307 by
 receptor associated tyrosine kinases effectively decreases the PP2A activity by inhibiting the
 interaction of PP2Ac with the PP2A-PR55/PR61 subunit [54]."
+SET Confidence = "Low"
 p(FPLX:PPP2, pmod(Ph,Tyr,307)) -| act(p(FPLX:PPP2))
+#p(FPLX:PPP2, pmod(Ph,Tyr,307)) -| complex(p(HGNC:PPP2R2A),p(HGNC:PPP2R5B))
+UNSET Confidence
 
+#annotation of the regulatory and core dimers
 SET Evidence = " The addition of a methyl group by LCMT1 at L309 enhances the binding affinity
 of the core dimer (A&C subunit) toward distinct regulatory subunits and provides specific
 activity to the holoenzyme [56]."
+SET Confidence = "Low"
 p(HGNC:LCMT1) -> p(FPLX:PPP2, pmod(M,Leu,309))
 p(FPLX:PPP2, pmod(M,Leu,309)) -> act(p(FPLX:PPP2))
+UNSET Confidence
 
 SET Evidence = " Akt plays an
 important role in cell growth, proliferation and apoptosis."
+SET Confidence = "High"
 p(FPLX:AKT) reg bp(GO:"cell growth")
 p(FPLX:AKT) reg bp(GO:"cell population proliferation")
 p(FPLX:AKT) reg bp(GO:"apoptotic process")
+UNSET Confidence
 
 SET Evidence = "Phosphorylation at Thr-308 and
 Ser-473 leads to activation of Akt and it was found that deregulation of Akt is associated
 with various human malignancies. "
+SET Confidence = "High"
 p(FPLX:AKT, pmod(Ph,Thr,308)) -> act(p(FPLX:AKT))
 p(FPLX:AKT, pmod(Ph,Ser,473)) -> act(p(FPLX:AKT))
+UNSET Confidence
 
+#core enzyme wih b55 together
 SET Evidence = "Co-immunoprecipitation and in-vitro pull down assay
 using pro-lymphoid FL5.12 cells showed a direct association of the PP2A-B55 holoenzyme
 with Akt, which selectively regulates phosphorylation of Akt at Thr-308 and regulates cell
 proliferation and survival [58]."
-p(FPLX:PPP2) -| p(FPLX:AKT, pmod(Ph,Thr,308))
+SET Confidence = "Low"
+p(FPLX:PPP2) reg p(FPLX:AKT, pmod(Ph,Thr,308))
 p(FPLX:PPP2) reg bp(GO:"cell population proliferation")
 p(FPLX:PPP2) reg bp(GO:"apoptotic process")
+UNSET Confidence
 
 SET Evidence = "PP2A also plays a major role in the Wnt signaling pathway. "
+SET Confidence = "High"
 p(FPLX:PPP2) -- bp(GO:"Wnt signaling pathway")
+UNSET Confidence
 
+#complex used
 SET Evidence = "In Xenopus, PP2A-
 B56 is involved in  β -catenin dephosphorylation and degradation and its phosphorylation
 directs activation of the Wnt pathway [43]."
+SET Confidence = "Low"
```

Comment:
see comment above
for B56 there are also different isoforms (alpha-epsilon)
maybe stick to the family of reulatory subunit b (FPLX:PPP2R_B)?
but its missing the specificity mentioned here...